astropy/modeling/functional_models.py

```diff
@@ -732,7 +732,7 @@ class Ellipse2D(Fittable2DModel):
     y_0 = Parameter()
     a = Parameter()
     b = Parameter()
-    theta = Parameter()
+    theta = Parameter(default=0)
```

Comment:
I think it's actually an oversight that this model didn't have defaults for all the parameters.  Could you also add defaults for the others while you're at it?  For the amplitude just 1, for the position 0, 0, and for the radii 1.